src/components/testimonials.jsx

```diff
@@ -1,35 +1,34 @@
+import Carousel from "react-material-ui-carousel";
+
 export const Testimonials = (props) => {
   return (
     <div id="testimonials">
       <div className="section-title text-center">
         <h2>What our clients say</h2>
       </div>
-      <div
-        className="container"
-        style={{ display: "flex", textAlign: "center" }}
-      >
-        <div className="row" style={{ margin: "auto" }}>
-          {props.data
-            ? props.data.map((d, i) => (
```

Comment:
If you were curious, doing the rendering this way breaks the carousel which is why I avoided doing this before. I changed to the conditional usage (`props?.data?.`) and it worked. Very strange but useful to know!

Reply:
props can be undefined/null/empty obj on first render which could cause it to break